commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce-input.scss

```diff
@@ -52,4 +52,12 @@
 			margin-bottom: 30px;
 		}
 	}
-}
\ No newline at end of file
+}
+
+.required-mark {
+	color: $color-mood-bad;
+
+	&::after {
+		content: '*';
+	}
+}
```

Comment:
trailing newline again :) please remove

Reply:
Hello @ethanbustad , I run the source formatter but it didn't change anything. Should I do it manually?

Reply:
done it, anyway

Reply:
Thanks, @FabioDiegoMastrorilli ! You're the best 🙇